packages/fx-core/templates/plugins/resource/hosting/bicep/webapp/config.template.bicep

```diff
@@ -0,0 +1,12 @@
+// Get existing app settings for merge
+var botCurrentAppSettings = list('${provisionOutputs.botOutput.value.botWebAppResourceId}/config/appsettings', '2021-02-01').properties
+
+// Merge TeamsFx configurations to Bot resources
+module teamsFxBotConfig '\{{PluginIdPlaceholder.Configuration.webapp.path}}' = {
```

Comment:
Since it is a app service bicep file, do we need put "Bot" concept in this file?